.github/workflows/auth-rate-limit-e2e-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Auth E2E (Rate-limit)
```

Comment:
Based on what I'm seeing it looks like if you were to run `npm run cypress:open --workspace=apps/authentication-service`, then it would show the rate limiting e2e tests as well as the express e2e tests. IMO we should set it up like it's set up for ucw-app. You have to run separate commands to open up the different suites. We don't want someone to think there's a failure because they don't have the correct configuration for the test to pass.